UPGRADING.md

```diff
@@ -13,6 +13,10 @@ The deprecated method `getCurrentPlatformEnum()` was removed from `SystemInfo` a
 The value returned from `macOSProcess` and `LinuxOSProcess` method `getCommandLine()` has been changed from null-delimited string to space-delimited string.
 Note : To parse the executables and arguments, the `getArguments()` method is the preferred alternative.
 
+### Changed the return type of `getServices()` in `OperatingSystem`
```

Comment:
This doesn't need a separate header here, just a new paragraph.  Also suggested wording: replace "to return" below with "now returns".